chain/polkadot/keys.go

```diff
@@ -7,7 +7,7 @@ import (
 
 	"github.com/decred/dcrd/dcrec/secp256k1/v2"
 
-	schnorrkel "github.com/ChainSafe/go-schnorrkel/1"
+	schnorrkel "github.com/ChainSafe/go-schnorrkel"
```

Comment:
this change caused issues with burnt / xion when we changed like this. Assume others will have issues too with the differences. So will recommend most not to update to this commit when issues arise

Reply:
was there some action you were looking for otherwise?